public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -125,6 +127,9 @@ export class DashboardModel {
     this.links = data.links || [];
     this.gnetId = data.gnetId || null;
     this.panels = _.map(data.panels || [], (panelData: any) => new PanelModel(panelData));
+    this.originalLibraryPanels = this.panels
+      .filter((panel) => panel.libraryPanel !== undefined)
+      .map((panel) => panel.libraryPanel!.uid!);
```

Comment:
think originalLibraryPanels is only used for unlink on save, so can remove this, is handled in backend (or should). so not sure we need this?